apps/dashboard/src/routes/org/[slug]/setup/+page.js

```diff
@@ -1,42 +1,11 @@
-export const load = ({ params = { slug: '' } }) => {
-  const data = [
-    {
-      title: 'Create Course',
-      desc: 'Create a course that you will share with your students',
-      isCompleted: true,
-      buttonLabel: 'Create Course'
-    },
-    {
-      title: 'Upload a profile picture and update username',
-      desc: 'Personalize and a human touch making interactions more personal and memorable',
-      isCompleted: false,
-      buttonLabel: 'Update Profile'
-    },
-    {
-      title: 'Update organisation profile picture',
-      desc: 'Establish a professional and recognizable identity for your organization',
-      isCompleted: true,
-      buttonLabel: 'Update Org Profile'
-    },
-    {
-      title: 'Create a lesson',
-      desc: 'Break your course into lesson that your students can easily understand',
-      isCompleted: false,
-      buttonLabel: 'Create Lesson'
-    },
-    {
-      title: 'Create an exercise',
-      desc: 'Test your students allow them to demonstarte their understanding of the subject matter',
-      isCompleted: false,
-      buttonLabel: 'Create Assignment'
-    },
-    {
-      title: 'Publish a course',
-      desc: 'Make your course public and purchaseable ',
-      isCompleted: false,
-      buttonLabel: 'Publish Course'
-    }
-  ];
+import { getSupabase } from '$lib/utils/functions/supabase';
+// import { getServerSupabase } from '$lib/utils/functions/supabase.server';
+const supabase = getSupabase();
+export const load = async ({ params = { slug: '' } }) => {
+  const { data } = await supabase
+    .from('onboarding_setup')
+    .select('*')
+    .order('id', { ascending: true });
```

Comment:
@Chifez Hey, you must note that you should get the `onboardin_setup` for a particular organization, not just all the `onboarding_setup` in the database.

Then you make a join to the organization table and filter by the `siteName`, which is the `slug` in this load function

Reply:
@Chifez Let's have a call tomorrow so we can discuss how to best implement this.

Reply:
> @Chifez Hey, you must note that you should get the `onboardin_setup` for a particular organization, not just all the `onboarding_setup` in the database.
> 
> Then you make a join to the organization table and filter by the `siteName`, which is the `slug` in this load function

oh yea that's true i need a reference_id in the onboarding_setup of the organization, then fetch all the content of the onbaording_setup table that matches this id, something like how you fetch courses, or better still completely restructure my onboading_setup table

Reply:
> @Chifez Let's have a call tomorrow so we can discuss how to best implement this.

yes please